src/routes/upload.ts

```diff
@@ -1,20 +1,29 @@
 import { UPLOAD_PATH } from '../config';
 import { Router } from 'express';
 import { createWriteStream, existsSync, rmSync } from 'fs';
+const { execSync } = require('child_process');
 
 const router = Router();
 
 router.post('/', (req, res) => {
   try {
     req.pipe(req.busboy); // Pipe it trough busboy
-
+    const fileSize = req?.query?.fileSize;
+    console.log(`From upload route: Uploaded File size: ${fileSize}`);
     req.busboy.on('file', (fieldname, file, data) => {
       const filename = data && data.filename ? data.filename : 'upload.raucb';
       console.log(`Upload of '${filename}' started`);
       // Create a write stream of the new file
       try {
         try {
           if (existsSync(UPLOAD_PATH + filename)) {
+            const existingFileSize = parseInt(execSync(`ls -l ${UPLOAD_PATH+filename} | awk '{print $5}'`));
```

Comment:
@HariHive this one looks slightly tricky,
let's maybe do it via promisified stat() function? Should look simpler and more readable